ci/credentials.example.yml

```diff
@@ -1,27 +1,29 @@
 pipeline-tasks-git-url: https://github.com/18F/cg-pipeline-tasks.git
 pipeline-tasks-git-branch: master
+
 cg-deploy-admin-ui-git-url: https://github.com/18F/cg-deploy-admin-ui.git
 cg-deploy-admin-ui-git-branch: master
 
-admin-ui-private-bucket-staging: SECURE_BUCKET_PRODUCTION
+admin-ui-private-bucket-staging: SECURE_BUCKET_STAGING
+admin-ui-staging-private-passphrase: SECURE_BUCKET_STAGING
+
 admin-ui-private-bucket-production: SECURE_BUCKET_PRODUCTION
+admin-ui-production-private-passphrase: SECURE_BUCKET_PRODUCTION
 
 admin-ui-staging-deployment-bosh-target: 192.0.2.99
 admin-ui-staging-deployment-bosh-username: admin
 admin-ui-staging-deployment-bosh-password: password
 admin-ui-staging-deployment-bosh-deployment: admin-ui
-admin-ui-staging-private-passphrase: passphrase
 
 admin-ui-production-deployment-bosh-target: 192.0.2.99
 admin-ui-production-deployment-bosh-username: admin
 admin-ui-production-deployment-bosh-password: password
 admin-ui-production-deployment-bosh-deployment: admin-ui
-admin-ui-production-private-passphrase: passphrase
 
-tf-state-bucket-staging:
-tf-state-file-staging:
-tf-state-bucket-production:
-tf-state-file-production:
+tf-state-bucket-staging: BUCKET_TERRAFORM_STAGING
```

Comment:
Is this clearer than before? It looks like the example values basically restate the key names.